index.html

```diff
@@ -163,67 +189,114 @@ <h2>Profesional Art <br>Printing Data</h2>
     </p>
-    <div class="grid">
+</div>
+    <div class="group"></div>
+</div>
 
     <div class="about-me">
         <button class="resume">Get my resume</button>
     </div>
     <div class="deco-img"></div>
     <hr>
     </div>
-        <ul class="myUL">
+    <div class="abilities">
+        <div>
+        <ul class="myUL" id="myUL-heading">
             <li><h2 class="language"><img src="images/Rectangle55.png" alt="diamond-shape" class="diamond">&nbsp Language</h2></li>
         </ul>
-        <ul class="myUL1"> 
+        <ul class="myUL2" id="myUL-li"> 
             <li class="lan">JavaScript</li>
             <li class="lan">Ruby</li>
             <li class="lan">HTML</li>
             <li class="lan">CSS</li>
             
         </ul>
-
-        <ul class="myUL-frameworks">
+          </div>  
+          <div>
+        <ul class="myUL-frameworks" id="myUL-frame">
             <li><h2 class="framework"><img src="images/Rectangle56.png" alt="diamond-shape" class="diamond">&nbsp Frameworks</h2></li>
         </ul>
-        <ul class="myUL1"> 
+        <ul class="myUL1 id="> 
             <li class="frame">React.js</li>
             <li class="frame">Ruby on Rails</li>
             <li class="frame">Rspec</li>
             <li class="frame">Capybara</li>
             
         </ul>
-
-        <ul class="myUL-skills">
+        </div>
+        <div>
+        <ul class="myUL-skills" id="myUL-sk">
             <li><h2 class="Skills"><img src="images/Rectangle57.png" alt="diamond-shape" class="diamond">&nbsp Skills</h2></li>
         </ul>
-        <ul class="myUL1"> 
+        <ul class="myUL1" id="myUL-li2"> 
             <li class="Skill">Database Management</li>
             <li class="Skill">Version Control</li>
             <li class="Skil">CLI</li>
             <li class="Skill">API Design</li>
             <li class="Skill">Web Development</li>
             
-        </ul>
-    
+        </ul> 
+        </div>
+        </div>   
+</section>
 
+<section class="contact">
+    <div class="contact-form">
+        <div>
+        <h3 class="contact-me">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        </div>
+    <div>
+        <form action="https://formspree.io/f/xgebyodb" method="POST">
+            <div class="Initials">
+            <input class="name-field" type="text" name="name" id="me" placeholder="First name" reuired="true" maxlength="30">
+            <input class="last-name" type="text" name="name" id="my" placeholder="Last name" reuired="true" maxlength="30">
+            </div>
+            <input class="email-field" type="email" name="email" id="mail" placeholder="Email address" reuired="true" >
+            <textarea class="text-area-field" name="area" id="text-area" rows="5" maxlength="500" placeholder="write me something..."></textarea>
```

Comment:
- [ ] Please ensure that all input fields have the required attribute to activate the validation on submission. This prevents the users from submitting the blank message ☺️.